cvat/schema.yml

```diff
@@ -9222,6 +9354,107 @@ components:
         compare_attributes:
           type: boolean
           description: Enables or disables annotation attribute comparison
+    RQJobDetails:
+      type: object
+      properties:
+        status:
+          allOf:
+          - $ref: '#/components/schemas/RQJobDetailsStatusEnum'
+          readOnly: true
+        message:
+          type: string
+          readOnly: true
+        id:
+          type: string
+          readOnly: true
+        operation:
+          $ref: '#/components/schemas/RQJobOperation'
+        percent:
+          type: number
+          format: double
+          readOnly: true
+          nullable: true
+        enqueue_date:
```

Comment:
I have just taken the API interface which was discussed and approved on HLD

Reply:
I will vote for `enqueued_date`.
In django admin we have `started/queued`. I am not sure this is the same time in case with RQ jobs. 
![image](https://github.com/cvat-ai/cvat/assets/40690378/f703857d-0d0e-4894-9c7b-eef103af4c97)